src/objects.rs

```diff
@@ -516,6 +516,11 @@ impl AddPrefix for Line {
     fn add_prefix(&mut self, prefix: &str) {
         self.id = prefix.to_string() + &self.id;
         self.network_id = prefix.to_string() + &self.network_id;
+    }
+}
+
+impl Line {
+    pub fn add_cm_prefix(&mut self, prefix: &str) {
         self.commercial_mode_id = prefix.to_string() + &self.commercial_mode_id;
```

Comment:
It's a bit hacky, if we had some commercial_mode to something else than Line I'm pretty sure that we will forgot to update this `cm_prefix ` thing.
I suggest to update `add_prefix` with the `cm_prefix`.
It's probably possible to do something way better with some Trait that every object that have a commercial_mode implements.

Reply:
cm prefix is now removed